packages/next-server/server/api-utils.ts

```diff
@@ -56,13 +59,34 @@ function parseJson(str: string) {
  * @returns Object with key name of query argument and its value
  */
 export function parseQuery({ url }: IncomingMessage) {
```

Comment:
probably better to name it `getQueryParser`, to match it with `getCookieParser`

Reply:
Could you make a suggestion please? 🙏